### PR TITLE
Adding the tools dashboard

### PR DESCRIPTION
**Pull request by jsignell**
Looks like:

Screen Shot 2019-06-14 at 6 08 24 PM

Still need to figure out ~~how to add nav~~ and whether or not to include so much CI info. 

Live at https://pyviz.github.io/website/tools.html

### CODE REVIEW

**Comment by jsignell** [comment]
This will be a squash merge...

File: tools/tools.yml [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+      badges: circleci, pypi, conda, site
+
+    - repo: pyviz/panel
+      site: panel.pyviz.org
+      badges: travis, appveyor, codecov, pypi, conda, site
+
+    - repo: QuantStack/voila
+      badges: travis, pypi, conda, rtd
+
+- name: Quick
```
Comment (jbednar) [inline tools/tools.yml]: I would say that all the ones in quick are high level, and vice versa.  What distinguishes them is maybe "High-Level Shared API" and "High-level"?

File: tools/tools.yml [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+    - repo: Kitware/VTK
+      conda_channel: conda-forge
+      badges: travis, pypi, conda, site
+
+    - repo: yt-project/yt
+      site: yt-project.org
+      badges: travis, codecov, pypi, conda, site
+
+- name: Content-area specific
```
Comment (jbednar) [inline tools/tools.yml]: Domain-specific?

**Comment by jsignell** [comment]
> Should we add a column for "Sponsor", i.e. NumFocus, Anaconda, Spotify, etc. ? 

Perhaps. That takes a little more detective work though and is less objective

**Comment by jbednar** [comment]
> > Should we add a column for "Sponsor", i.e. NumFocus, Anaconda, Spotify, etc. ?
>
> Perhaps. That takes a little more detective work though and is less objective

Here's the detective work done:

```
bokeh           NumFocus (+ Anaconda? +Microsoft?)
matplotlib      NumFocus
plotly.py       Plot.ly

dash            Plot.ly
panel           Anaconda
voila           QuantStack (+ Bloomberg?)

chartify        Spotify
holoviews       Anaconda
seaborn         N/A
plotly_express  Plot.ly
altair          N/A

pdvega          N/A
hvplot          Anaconda
PandasBokeh     N/A
cufflinks       N/A

vispy           N/A
glumpy          N/A
mayavi          Enthought
ParaView        Kitware
VTK             Kitware
yt              NumFocus

networkx        N/A
scikitimage     N/A
cartopy         Met Office
geoviews        Anaconda

dask            Anaconda (+ NVidia?)
pandas          NumFocus
xarray          NumFocus
colorcet        Anaconda
datashader      Anaconda

cmocean         N/A
viscm           N/A
vaex            vaex.io
```

As for objectivity, I think the first sponsors listed above are quite unambiguous; those are definitely sponsors, and there are sponsors for enough of the libraries that it's helpful to have that information.  There are two less well-defined aspects, though:

1. Most of the ones marked N/A have an individual developer as the single main author.  We could list that author in this column, but I'm not sure that's appropriate; everything else also has at least one author, and it's not the same thing as having a sponsor.  Authorship varies over time, more than sponsorship does, and how much it's run by one person and how much by a community also varies a lot; pinning down both those things seems very tricky. So my vote is to track only institutional sponsors, as that's fairly objective.

2. I'm not sure what's best for the cases where there are multiple institutional sponsors:
    * **Voila** is funded by Bloomberg to some extent, but it's very much being run by QuantStack, so I'm inclined to say it should just be QuantStack unless QuantStack or Bloomberg request otherwise.
    * **Dask** was originally an Anaconda project, but the original main author is now at Nvidia and there are people paid to work on Dask at both companies. There's a good argument for listing just Anaconda (where the project was created and managed for many years), or both (to declare which companies are putting resources into this project on a sustained basis).  Both options seem reasonable to me, for Dask.  
    * **Bokeh** was originally an Anaconda project, and most people funded to work on it are at Anaconda, but like Dask the main maintainer is now at a different company, Microsoft.  I don't know the details of Microsoft's involvement, but I don't think they are contributing significant funds and thus I would only list them as a sponsor if someone from Bokeh or Microsoft requests that.  In any case, Bokeh is now run by a NumFocus-appointed committee, and so I'm inclined to list NumFocus as the main sponsor, with Anaconda listed as well if it's ok to list multiple sponsors.

**Comment by jsignell** [comment]
1. I agree the institutional sponsors matter more. If I were to try to list authors I'd want to go with something like top contributor for each project. 

2. I think we should be very flexible with this field and open to any suggestions.  Multiple sponsors should definitely be allowed, and we should just do a good-faith effort to do what we think is right.

**Comment by jbednar** [comment]
Sounds good.  I think the main use of this page is to help people understand the projects and what they can do for them, and so I think we should omit the top contributor info, which isn't really going to help people make that decision.  We could still add it to recognize the authors, but I would think that's up to each project to decide what to do, not us.

Ok, if you can see how to add multiple sponsors in the table format, great!

**Pull request merged**